week-3/todo-list/index.html

```diff
@@ -5,14 +5,18 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Title here</title>
     <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css" integrity="sha512-DTOQO9RWCH3ppGqcWaEA1BIZOC6xxalwEsw9c2QQeAIftl+Vegovlnee1c9QX4TctnWMn13TZye+giMm8e2LwA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+
   </head>
   <body>
     <form>
       <div>
-        <input type="text" placeholder="New todo..." />
+        <input  id="input-todo" type="text" placeholder="New todo..." />
+        <button  id="add-button" type="submit">Add Todo</button>
       </div>
       <div>
-        <button type="submit">Add Todo</button>
+        <input  id="date" type="date" />
```

Comment:
What is the purpose of the date object? As a user what should I expect to happen when I select or change the date input? Does it have any affect on the Todo list item?

Reply:
Thank you, @nirmeet-baweja, for reviewing my code. I intended for users to be able to select a deadline for their tasks using the date object, but it's not working in my program, and I'm stuck fixing it.

Reply:
Could you please elaborate?